prep_inputs_for_combiner.py

```diff
@@ -0,0 +1,301 @@
+    'duplicate_metrics': 'duplicate_metrics',
+    'insert_size_metrics': 'insert_size_metrics',
+    'wgs_metrics': 'wgs_metrics',
+}
+
+
+def _find_gcs_files(warp_executions_bucket, work_dir, file_suffix, file_key):
+    safe_mkdir(work_dir)
+    found_file_fpath = join(work_dir, f'found_{file_key}.txt')
+    if not isfile(found_file_fpath):
+        path_tmpl = join(warp_executions_bucket, '**', f'*.{file_suffix}')
+        run_cmd(f'gsutil ls \'{path_tmpl}\' > {found_file_fpath}')
+    with open(found_file_fpath) as f:
+        found_files = [l.strip() for l in f]
+    found_path_by_sname = {
+        os.path.basename(fp).replace(f'.{file_suffix}', ''): fp
+        for fp in found_files
+    }
+    return found_path_by_sname
+
+
+def _randomise_pop_labels(sample_df):
+    # Adding population labels for 2/3 of the samples in a group;
+    # the rest will be used to test the ancestry inferring methods
+    return list(
+        sample_df\
+            .groupby(['Population']) \
+            .apply(lambda x: x.iloc[:int(x['Population'].size / 1.5)]) \
+            ['Individual.ID']
+    )
+
+
+def _move_locally(gvcf_by_sample, dataset, picard_file_by_sname_by_key):
+    local_gvcf_by_sample = dict()
+    local_picard_file_by_sname_by_key = defaultdict(dict)
+    for sample, gvcf_path in gvcf_by_sample.items():
+        local_path = f'gs://cpg-fewgenomes-upload/{sample}/gvcf/{basename(gvcf_path)}'
+        if not file_exists(local_path):
+            run_cmd(f'gsutil cp {gvcf_path} {local_path}')
+        if not file_exists(local_path + '.tbi'):
+            run_cmd(f'gsutil cp {gvcf_path}.tbi {local_path}.tbi')
+        local_gvcf_by_sample[sample] = local_path
+
+        for picard_key, picard_path_by_sname in picard_file_by_sname_by_key.items():
+            picard_path = picard_path_by_sname.get(sample)
+            if picard_path:
+                local_path = f'gs://cpg-fewgenomes-upload/{sample}/picard_files/' \
+                             f'{basename(picard_path)}'
+                if not file_exists(local_path):
+                    run_cmd(f'gsutil cp {picard_path} {local_path}')
+                local_picard_file_by_sname_by_key[picard_key][sample] = local_path
+    return local_gvcf_by_sample, local_picard_file_by_sname_by_key
+
+
+@click.command()
+@click.option(
+    '--dataset',
+    'dataset_name',
+    required=True,
+    help='Dataset name, e.g. "50genomes". Assumes that '
+         '`{datasets_dir}/{datasets_name}/samples.ped` exists, unless --ped file is '
+         'specified explicitly.'
+)
+@click.option(
+    '--ped',
+    'samples_ped',
+    help='Ped file with input sample names. If not provided, '
+         '`{datasets_dir}/{datasets_name}/samples.ped` is read.'
+)
+@click.option(
+    '--warp-executions-bucket',
+    'warp_executions_bucket',
+    default=('gs://playground-us-central1/cromwell/executions/'
+             'WGSMultipleSamplesFromBam'),
+    help='Bucket with WARP workflow outputs'
```

Comment:
Sorry for nitpicking:

```suggestion
    help='Bucket with WARP workflow outputs.'
```